package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Exercises for JS1 Week 1",
   "license": "CC-BY-SA-4.0",
   "scripts": {
-    "test": "jest"
+    "test": "jest","extra-tests":"jest extra"
```

Comment:
You don't need to change this file.
If you run `npm test` it runs all the tests for you anyways.